### ISSUE TITLE
Not working with new julia 1.3

### ISSUE THREAD

**Issue by doc4child** [RandomString123/FWF.jl#22]
The package does not install, also, if I tried to do url based install than since it does not handle missing as dependency, it errors out.

**Comment by bkamins** [comment]
CSV.jl supports FWF format now - you could try it.

**Comment by DimitarVanguelov** [comment]
@bkamins CSV.jl appears to only support fixed-width files by setting `delim=' '` and `ignorerepeated=true`, unless I missed something. That is not robust enough for all use cases, such as having a column that contains a space. This was discussed a bit [here](https://discourse.julialang.org/t/reading-fixed-width-files/30163/6?u=earthgoddessdude). 

In that thread, I timidly offered to write something, but doing a PR to one of the major text-parsing packages really does seem beyond my abilities at the moment (especially given my work/life balance). Passing an array of tuples or unit ranges really is a nice feature that seems to be currently missing from CSV.jl or TableReader.jl. The next best thing is a package like FWF.jl, which seems to already have that feature. 

I know you've been heavily involved with DataFrames (I accidentally signed up for notifications, and my email's been blowing up lately 😂), but is there any chance of this package being brought back to life?

**Comment by bkamins** [comment]
Agreed CSV.jl has a limited functionality and something dedicated would be useful.

@RandomString123 - can you please comment what are your plans regarding this package?

**Comment by RandomString123** [comment]
Unfortunately, at this time I am busy at work and don't have the free time to continue this project.  I could either transfer it to one of you.  Or you could fork it and i'll make this repo read only with a pointer to the new repo.

**Comment by DimitarVanguelov** [comment]
Hi @RandomString123 

I actually forked it last night so I can play around with it and see if I can make it v1.0+ compatible. I was intending on submitting a pull request if I ever got it working. 

My concern about making a pointer to another repo is that there is already another FWF.jl repo (by @bkamins ) that’s deprecated and pointing to this one. 

Unless @bkamins wants it, I can take this on, with the disclaimer that I am relatively new to programming and also busy with work. But I am also eager to get practice and make a contribution for once.

**Comment by bkamins** [comment]
I am OK if you take over.

**Comment by RandomString123** [comment]
I am fine with it also.  I think this repo was the same "learning" for bkamins and myself.  Eventually i'd like to get back into Julia I am stuck in Python hell right now.  

In either case i'll add you as a contributor to this repo which should give you almost full control over it.

**Comment by DimitarVanguelov** [comment]
Thanks! I'll try not to let you down 😅. Will either of you be occasionally available for questions/advice?
